root/report/PlacesWithoutCoordinates.js

```diff
@@ -66,7 +68,7 @@ const PlacesWithoutCoordinates = ({
                 <td>{item.place.address}</td>
                 <td>
                   {item.place.area
-                    ? <EntityLink entity={item.place.area} />
+                    ? <DescriptiveLink entity={item.place.area} />
```

Comment:
Is https://github.com/metabrainz/musicbrainz-server/pull/931/commits/a16b759572eaae9b07cd3cdbbdf016f99f487416 supposed to show area’s parents or something? I cannot see any difference.

Reply:
Yes. It used to work, but I'll re-check in case it now doesn't.

Reply:
Alright, areas are just not linked to countries in sample data.